spec/test_spec.rb

```diff
@@ -0,0 +1,94 @@
+require './book'
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './rental'
+
+describe Book do
+  context 'When testing the Book class' do
+    it 'Should show the Title of the book' do
+      book = Book.new('Foundation', 'Isaac Asimov')
+      expect(book.title).to eq 'Foundation'
+    end
+    it 'Should show the Author of the book' do
+      book = Book.new('Foundation', 'Isaac Asimov')
+      expect(book.author).to eq 'Isaac Asimov'
+    end
+  end
+end
```

Comment:
- [ ] It would be better if we have one file per class. For example; the `book` class should have its' own file, like `spec/book_spec.rb` in this form. This will help tracking bugs easier and faster